test/parallel/test-http-server-unconsume-consume.js

```diff
@@ -4,8 +4,8 @@ const http = require('http');
 
 const testServer = http.createServer(common.mustNotCall());
 testServer.on('connect', common.mustCall((req, socket, head) => {
-  socket.write('HTTP/1.1 200 Connection Established' + '\r\n' +
-      'Proxy-agent: Node-Proxy' + '\r\n' +
+  socket.write('HTTP/1.1 200 Connection Established\r\n' +
+      'Proxy-agent: Node-Proxy\r\n' +
```

Comment:
I wonder why the new eslint rule didn't complain about the next `'\r\n'` not being included as well?
I also wonder if the rule takes into account max line length rules...